### ISSUE TITLE
Parse error for macro map_for

### ISSUE THREAD

**Issue by jeberger** [lukaslueg/macro_railroad#9]
When pasting the code for the [`map_for` macro](https://bitbucket.org/jmb/rust-map_for/src/default/src/lib.rs#lines-366) into the [wasm demo](https://lukaslueg.github.io/macro_railroad_wasm_demo/), I get:

```
Failed to parse, and I didn't even write an error-handler. Anyway:

ParseError(None)
```

However, the same version of `map_for` [works with the latest Rust compiler (v1.28.0)](https://ci.appveyor.com/project/jmb/rust-map-for/build/job/nwtcxs58t727gdya).

**Comment by dtolnay** [comment]
The macro uses `$move` as a fragment variable so this looks like the same problem as https://github.com/lukaslueg/macro_railroad/issues/5.

**Comment by lukaslueg** [comment]
Fixed in 2a45b0c
